spec/services/db/monthly_sp_auth_count/unique_monthly_auth_counts_by_iaa_spec.rb

```diff
@@ -151,7 +157,7 @@
             year_month: '202010',
             iaa_start_date: iaa_range.begin.to_s,
             iaa_end_date: iaa_range.end.to_s,
-            total_auth_count: 300,
+            total_auth_count: 21,
```

Comment:
now that we're using the direct table, this was just to create fewer rows and make a faster test